dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/runner/AlertSender.java

```diff
@@ -78,17 +78,17 @@ public void run() {
             AlertInfo alertInfo = new AlertInfo();
             alertInfo.setAlertData(alertData);
 
-            alertInfo.addProp("receivers", receviersList);
+            alertInfo.addProp("receivers", receiversList);
 
             AlertPlugin emailPlugin = pluginManager.findOne(Constants.PLUGIN_DEFAULT_EMAIL_ID);
             retMaps = emailPlugin.process(alertInfo);
 
             if (retMaps == null) {
                 alertDao.updateAlert(AlertStatus.EXECUTION_FAILURE, "alert send error", alert.getId());
-                logger.info("alert send error : return value is null");
+                logger.error("alert send error : return value is null");
             } else if (!Boolean.parseBoolean(String.valueOf(retMaps.get(Constants.STATUS)))) {
                 alertDao.updateAlert(AlertStatus.EXECUTION_FAILURE, String.valueOf(retMaps.get(Constants.MESSAGE)), alert.getId());
-                logger.info("alert send error : {}", retMaps.get(Constants.MESSAGE));
+                logger.error("alert send error : {}", retMaps.get(Constants.MESSAGE));
             } else {
```

Comment:
This log level, please trouble confirm it again. @dailidong

Reply:
Looks good to me